source/UnexpectedInExpected.bs

```diff
@@ -80,15 +83,13 @@ Impact on the standard {#impact}
 
 This change is entirely based on library extensions and does not require any language features beyond what is available in C++ 23.
 
-
 Wording {#word}
 =======
 
 <b>TODO:</b> This is the fun part. We need to add diffs to current draft of the standard here
 
-
 References {#ref}
 ==========
 
-* <a href="https://wg21.link/P0032r3">P0032</a> Homogeneous interface for variant, any and optional
-* <a href="https://wg21.link/P0323r12">P0323</a> std::expected
+* <a href="https://wg21.link/P0032">P0032</a> Homogeneous interface for variant, any and optional
```

Comment:
Bad idea to change the link - wg21.link does not necessarily gives the correct latest revision, hence i specified by hand